packages/blocks/src/__internal__/utils/rect.ts

```diff
@@ -1,3 +1,5 @@
+import { clamp } from '../index.js';
+
 export class Point {
```

Comment:
I think we need unit test for the basic class

Reply:
please use vitest with happy-dom

Reply:
This class implementation looks trivial, we have added tests in `tl-utils` in phasor. So I suggest not to do that in this PR.